packages/codebytes/babel.config.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  presets: ['codecademy', '@babel/preset-typescript'],
+  plugins: [
+    [
+      '@emotion',
+      {
+        sourceMap: true,
+        autoLabel: 'always',
+        labelFormat: '[local]',
+      },
+    ],
+  ],
+  include: ['./src/**/*'],
+  ignore: ['__tests__', './**/*.d.ts'],
```

Comment:
this file is copied directly from gamut for local packages. it also ensures that the compiler recognizes typescript